### ISSUE TITLE
Add Protobuf to Data Import docs

### ISSUE THREAD

**Issue by mradamcox** [GeoDaCenter/webgeoda#12]
TL;DR: Is it possible for me to contribute to the [GitBook documentation](https://docs.webgeoda.org) for WebGeoDa?

Hi! I'm working with the WebGeoDa codebase that was deployed for the Opioids Environment Policy Scan (OEPS) project, currently hosted at https://oeps.healthyregions.org. I really appreciate the detailed documentation you've made for WebGeoDa Scaffolding, and was wondering if/how I can contribute to it.

The OEPS project uses a few external vector tilesets that are stored in Mapbox, which I assume is the Protobuf capability described at the bottom of [this page](https://docs.webgeoda.org/data/importing-data). I would be happy to add content to fill the advanced docs gaps (to the best of my abilities), as the current link on that page is broken--seems like the `nofurtherinformation/webgeoda-gitbook` repo has been removed.

Thank you!
